src/main/java/org/gridsuite/modification/server/entities/equipment/creation/VscCreationEntity.java

```diff
@@ -26,6 +26,7 @@
 @Getter
 @Entity
 @Table(name = "vscCreation")
+@PrimaryKeyJoinColumn(foreignKey = @ForeignKey(name = "vscCreation_id_fk_constraint"))
```

Comment:
to remove ? 
No residual changeset about this PK and FK

Reply:
Do later, to remove